validator/src/main/resources/expected-data-template/otelSDKexpectedAWSSDKTrace.mustache

```diff
@@ -1,58 +1,37 @@
 [{
-  "name": "/aws-sdk-call",
+  "name": "127.0.0.1:8080",
```

Comment:
Do you think all SDKs will generate this name? We should just use the env variable to set the service name so that top level segment name can be similar within all SDKs

Reply:
I'm actually not sure, I think this service name makes sense _if_ the sample app has a backend service that is instrumented to create a span with a `http.host` attribute, but if it doesn't, this will break.

In this specific case, the `{{environment}}` variable has `http://127.0.0.1:8080` so it will fail to match `127.0.0.1:8080` 😕 

I think if we count on there being `http.request.url == {{endpoint}}/aws-sdk-call` then we can count on there being a service name of `127.0.0.1:8080` because that _is_ the service. So in that case, it would be _ok_ to leave this.

Otherwise we might consider removing the `"name"` check if the SDK _does not_ add `http.host` or even if it adds `service.name` instead. What do you think?

Reply:
We spoke about it and decided that for now, we should remove the `"name"` property until we establish something consistent across the SDKs.